### ISSUE TITLE
FORMViewController

### ISSUE THREAD

**Issue by 3lvis** [hyperoslo/Form#255]
Initializing a new UICollectionViewController that uses Form involves one too many steps.

The UICollectionViewLayout part, it doesn't feel natural, it feels weird.

https://github.com/hyperoslo/form-bug-219/blob/master/Pod/ViewController.m#L13-L25

I was thinking it could be great if we had a FORMViewController that you could inherit from and then you'll have a predefined initializer.

So you'll do

```objc
FORMSubclassViewController *controller = [[FORMSubclassViewController alloc] initWithJSON:JSON
                                                                                   values:nil
                                                                                 disabled:NO];
```

**Comment by NSElvis** [comment]
It can be a good example on how to create a UICollectionViewController that uses Form.

**Comment by 3lvis** [comment]
Fixed in #392
